scripts/bundleSize/bundleSizeConfig.js

```diff
@@ -1,13 +1,11 @@
-module.exports = {
-  /** Size limit for all bundles used by each service (kB)
-   *
-   * `MIN_SIZE` should be the value of "Smallest total bundle size (kB) (smallest service + smallest page)" &
-   * `MAX_SIZE` should be the value of "Largest total bundle size (kB) (largest service + largest page)"
-   * from the "MODERN service config & theme + page bundle sizes summary" in the build output
-   *
-   * We are allowing a variance of -5 on `MIN_SIZE` and +5 on `MAX_SIZE` to avoid the need for frequent changes, as bundle sizes can fluctuate
-   */
+export const MIN_SIZE = 783 - 5;
+export const MAX_SIZE = 1238 + 5;
```

Comment:
Can we move these values to below the comment (it was there to explain why it's like this, and what devs would need to update to increase the bundle size if necessary)